src/components/button/button.tsx

```diff
@@ -129,16 +154,29 @@ export class Button implements LabelableComponent, InteractiveComponent, FormOwn
     }
   }
 
+  @Watch("intlLoading")
+  @Watch("defaultMessages")
+  @Watch("messageOverrides")
+  onMessagesChange(): void {
+    /** referred in t9n util */
+  }
+
   //--------------------------------------------------------------------------
   //
   //  Lifecycle
   //
   //--------------------------------------------------------------------------
 
-  connectedCallback(): void {
+  async connectedCallback(): Promise<void> {
+    if (Build.isBrowser) {
```

Comment:
Why do we need to check for `isBrowser` here?

Reply:
Invoke `mutationObserver` only in the browser context and avoid hydrate errors for prerender builds. This is an interesting case where this occurs only for a few components.

Reply:
it is resolved now. we longer need to check for build context at the component level.